### DISCUSSION TITLE
[A-L] (20 Settembre 2024) Esercizio 2

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#418] [Esami A-L]
Siano $\Large f : \mathbb{N} \rightarrow \mathbb{N}$ e $\Large g : \mathbb{N}\ \textbackslash\ \\{2\\} \rightarrow \mathbb{N}$ due funzioni.

Indicare se le seguenti affermazioni sono vere o false.

1. Se f è suriettiva allora g non è la sua inversa (ossia $\Large f^{-1} \neq g$). [1 punto]

2. È possibile che per ogni $\Large n \in \mathbb{N},\ (g \circ f)(n) = n$. [2 punti]

3. Se g è la funzione identità allora lo è anche $\Large (g \circ f)$. [2 punti]

**Comment by cek753** [comment]
1 Vero
2 Vero
3 Falso, non è definita per f(n)=2
